sentry-android-distribution/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+  <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
I think it's fine to keep this either way but I was wondering - the requirement is you have to have the main Sentry SDK right? You won't be able to have just distro.